spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
+
+    it "splits the lines correctly" do
+      message = "hello\nworld\n"
```

Comment:
you should do it, as it give clarity ...

why not having something like:

```let(:separator)```
```let(message)```

then in every context you redefine the separator. (just an idea, as I said just styling, but we should DRY us from the beginning ) ;-)